DirectProgramming/DPC++FPGA/Tutorials/Features/memory_attributes/README.md

```diff
@@ -29,17 +29,17 @@ Memory attributes can be applied to any variable or array defined within the ker
-| intelfpga::simple_dual_port      | Specifies that the memory implementing the variable or array should have no port that services both reads and writes.
-| intelfpga::merge("`key`", "`type`")  | Merge two or more variables or arrays in the same scope width-wise or depth-wise. All variables with the same `key` string are merged into the same memory system. The string `type` can be either `width` or `depth`. 
-| intelfpga::bank_bits(b<sub>0</sub>,b<sub>1</sub>,...,b<sub>n</sub>)  | Specifies that the local memory addresses should use bits (b<sub>0</sub>,b<sub>1</sub>,...,b<sub>n</sub>) for bank-selection, where (b<sub>0</sub>,b<sub>1</sub>,...,b<sub>n</sub>) are indicated in terms of word-addressing. The bits of the local memory address not included in (b<sub>0</sub>,b<sub>1</sub>,...,b<sub>n</sub>) will be used for word-selection in each bank. 
+| `intel::fpga_register`           | Forces a variable or array to be carried through the pipeline in registers.
+| `intel::memory("impl_type")`     | Forces a variable or array to be implemented as embedded memory. The optional string parameter `impl_type` can be `BLOCK_RAM` or `MLAB`.
+| `intel::numbanks(N)`             | Specifies that the memory implementing the variable or array must have N memory banks. 
+| `intel::bankwidth(W)`            | Specifies that the memory implementing the variable or array must be W bytes wide.
+| `intel::singlepump`              | Specifies that the memory implementing the variable or array should be clocked at the same rate as the accesses to it.
+| `intel::doublepump`              | Specifies that the memory implementing the variable or array should be clocked at twice the rate as the accesses to it.
+| `intel::max_replicates(N)`       | Specifies that a maximum of N replicates should be created to enable simultaneous reads from the datapath.
+| `intel::private_copies(N)`       | Specifies that a maximum of N private copies should be created to enable concurrent execution of N pipelined threads.
+| `intel::simple_dual_port`        | Specifies that the memory implementing the variable or array should have no port that services both reads and writes.
+| `intel::merge("key", "type")`    | Merge two or more variables or arrays in the same scope width-wise or depth-wise. All variables with the same `key` string are merged into the same memory system. The string `type` can be either `width` or `depth`. 
+| `intel::bank_bits`(b<sub>0</sub>,b<sub>1</sub>,...,b<sub>n</sub>)  | Specifies that the local memory addresses should use bits (b<sub>0</sub>,b<sub>1</sub>,...,b<sub>n</sub>) for bank-selection, where (b<sub>0</sub>,b<sub>1</sub>,...,b<sub>n</sub>) are indicated in terms of word-addressing. The bits of the local memory address not included in (b<sub>0</sub>,b<sub>1</sub>,...,b<sub>n</sub>) will be used for word-selection in each bank. 
```

Comment:
It's probably better to do `intel::bank_bits(b0, b1, ..., bn)` and change the two bit lists in the description accordingly.